sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/utils.py

```diff
@@ -295,6 +299,22 @@ def _is_input_or_output_type(io: type, type_str: str):
                         )
             except BaseException as e:
                 raise UserErrorException(f"Failed to parse {io} to azure-ai-ml Input/Output: {str(e)}") from e
+                # Handle Annotated annotation
+        elif isinstance(io, _AnnotatedAlias):
```

Comment:
Double check if there is a public method to check the type.

Reply:
done